_posts/2017-06-23-release-0.3p5.md

```diff
@@ -0,0 +1,83 @@
+- **file** - Provides facilities for accessing the filesystem. This library can only be used when Eve is running in headless mode.
+- **console** - Allows you to write messages to console log, warn, and error.
+
+## Changelog
+
+### Platform
+
+We made a couple fixes to the Eve platform:
+
+- Action lookups in the syntax now support dynamic attributes - [[849](https://github.com/witheve/Eve/pull/846)]
+- Fixed an issue with aggregates and grouping, which would cause blocks using them to fail - [[856](https://github.com/witheve/Eve/pull/856)]
+- Fixed unterminated record/quote errors - [[848](https://github.com/witheve/Eve/pull/848)]
+- Prevent exception from program which parses no code blocks - [[850](https://github.com/witheve/Eve/pull/850)]
+- Fixed double-printing first line on error spans that start at the beginning of the block - [[848](https://github.com/witheve/Eve/pull/848)]
+- Better error message for non-fixpoint - [[847](https://github.com/witheve/Eve/pull/847)]
+- Add a custom error message for values without attributes in a record - [[845](https://github.com/witheve/Eve/pull/845)]
+- Allow appendAsEAVs to also handle nested sub-records - [[861](https://github.com/witheve/Eve/pull/861)]
+- Export all first party watcher classes - [[862](https://github.com/witheve/Eve/pull/862)]
+
+### Renderer
+
+- Enables using `html.addExternalRoot()` to control SVG placement - [[846](https://github.com/witheve/Eve/pull/846)]
+- Enables mixing SVG into HTML documents - [[846](https://github.com/witheve/Eve/pull/846)]
+
+### Library
+
+For the first time, the standard library now has a baseline of [documentation](http://docs.witheve.com/v0.3/handbook/libraries/stdlib/). The documented libraries include aggregates, math, strings, HTML, canvas, UI, system, file, and console.
+
+We've also changed/added the following:
+
+- added the [`math/pow[]`](https://github.com/witheve/Eve/blob/master/src/runtime/stdlib.ts#L191) function - [[856](https://github.com/witheve/Eve/pull/856)]
+- added the [`math/convert-base[]`](https://github.com/witheve/Eve/blob/master/src/runtime/stdlib.ts#L211) function - [[856](https://github.com/witheve/Eve/pull/856)]
+- added the `#notify` library - [[859](https://github.com/witheve/Eve/pull/859)]
+- added the `#file` library - [[858](https://github.com/witheve/Eve/pull/858)]
+- added the `#console` library - [[858](https://github.com/witheve/Eve/pull/858)]
+- ([BREAKING](https://groups.google.com/forum/#!topic/eve-talk/SF_ittD0nU8)) - adjusted the attributes on [`#system/timer`](http://docs-next.witheve.com/v0.3/handbook/libraries/stdlib/#system):
+    - fixed `day` attribute on timer to report the day of the month, not the day of the week - [[853](https://github.com/witheve/Eve/pull/853)]
+    - added `weekday` and `millisecond` attributes - [[853](https://github.com/witheve/Eve/pull/853)]
+    - changed `frames` to `tick` - [[855](https://github.com/witheve/Eve/pull/855)]
```

Comment:
Would explicitly mention that we depluralized seconds, minutes and hours here inline.